CLAUDE.md

```diff
@@ -0,0 +1,128 @@
+# EMG API v2 - Claude Development Guide
+
+## Project Overview
+
+This is a Django-based bioinformatics application for MGnify (formerly EBI Metagenomics) that orchestrates large-scale genomic analysis pipelines. The system manages studies, samples, assemblies and analyses with sophisticated workflow orchestration.
+
+## Architecture
+
+### Core Apps
+- **`analyses/`** - Core MGnify domain models (Study, Sample, Run, Assembly, Analysis)
+- **`ena/`** - European Nucleotide Archive integration (Study, Sample models)
+- **`workflows/`** - Bioinformatics pipeline orchestration using Prefect
+
+### Key Domain Models
+```
+Study (MGYS*) ← Analysis (MGYA*) ← Sample ← Run/Assembly
```

Comment:
maybe I'm dumb but I thought it was sample -> run/assembly rather than the other way around?